internal/services/kusto/kusto_attached_database_configuration_resource.go

```diff
@@ -85,6 +85,63 @@ func resourceKustoAttachedDatabaseConfiguration() *pluginsdk.Resource {
 					string(kusto.DefaultPrincipalsModificationKindUnion),
 				}, false),
 			},
+
+			"sharing_properties": {
```

Comment:
i don't think properties makes sense here?
```suggestion
			"sharing": {
```